libs/accounts/email-renderer/src/templates/verifyLoginCode/index.ts

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export type TemplateData = {
+  code: string;
+  passwordChangeLink: string;
+  serviceName: string;
+  time: string;
+  device: {
+    uaBrowser: string;
+    uaOSVersion: string;
+    uaOS: string;
+  };
+  location: {
+    stateCode: string;
+    country: string;
+    city: string;
```

Comment:
Should some of these types for partials, like `location` mapping to `partials/userLocation/index.mjml` in auth-server`, go in their own type that we can import and use?

It looks like at least `city` and `stateCode` should be optional, based on the template if/elses.

Reply:
This might be a limitation of the AI extraction I used to generate these types. This is a good call out though. If you see other stuff like this, let me know, and I'll do a fast follow for cleaning up repetitive types.

Reply:
Might be worth looking at what all is in that `/partials` directory? The two I called out stood out to me since I'd looked at them recently.